tests/unit/api/badge/test_badge_common.py

```diff
@@ -0,0 +1,63 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from http import HTTPStatus
+
+from flask import current_app
+from unittest.mock import MagicMock
+
+from metadata_service import create_app
+from metadata_service.api.badge import BadgeCommon
+from metadata_service.entity.resource_type import ResourceType
+from tests.unit.api.dashboard.dashboard_test_case import DashboardTestCase
+
+BADGE_NAME = 'alpha'
+CATEGORY = 'table_status'
+BADGE_TYPE = 'neutral'
+
+
+class TestDashboardBadgeAPI(DashboardTestCase):
```

Comment:
this should be TestBadgeCommon ?

Reply:
I followed the Tag tests which use dashboard to test tag common so that's why the naming is a bit confusing